packit/cli/create_update.py

```diff
@@ -61,11 +61,25 @@
     required=False,
     default="enhancement",
 )
+@click.option(
+    "--bugzilla-id",
+    help="Bugzilla ID that is resolved with the update",
+    required=False,
+    default=None,
+    type=click.types.INT,
+    multiple=True,
```

Comment:
ad naming: what about `-b/--resolve-bugzilla`?
ad chaining: yes, twice… idea is that they are integers, so it is easier to validate them via `click` with `INT` type and being able to have multiple of those rather than „eating“ string from the user that would fail later on in API request :shrug: However I'm not against :)

Also going to check if you can do both via comma with current implementation.

Reply:
Yeah, I can't… I can do a custom validation of the option though :)